backend/app/views/spree/admin/shared/_content_header.html.erb

```diff
@@ -1,23 +1,13 @@
 <% if content_for?(:page_title) || content_for?(:page_actions) %>
   <div id="content-header" data-hook>
-    <div class="container">
-      <div class="sixteen columns">
-        <div class="block-table">
-          <% if content_for?(:page_title) %>
-            <div class="table-cell">
-              <h1 class="page-title <%= yield :page_title_classes %>"><%= yield :page_title %></h1>
-            </div>
-          <% end %>
+    <h1 class="page-title <%= yield :page_title_classes %>"><%= yield :page_title %></h1>
```

Comment:
Do you know what `page_title_classes` is used for? Not sure if we want to keep this around. It doesn't make a lot of sense to me.